### PR TITLE
Lesson 1 6

### PR DESCRIPTION
**Pull request by Victoria240**
React Fundamentals: Reusable Components, Imperative React

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, {useRef, useEffect} from 'react'
+
+function InputWithLabel({ children, value, onChange}) {
+    const inputRef = useRef(null)
+
+    useEffect(() => {
+         // Accessing the input element and focusing it
+        inputRef.current.focus();
+    }, []);
```
Comment (dargaCode) [inline src/InputWithLabel.js]: This is very unintuitive but having an empty array for the dependency array argument triggers the effect only on the initial render. In this case we want to trigger the effect on *every* render. To support that we want no dependency array argument at all (leaving it `undefined`)

```suggestion
    });
```

Comment (dargaCode) [reply src/InputWithLabel.js]: after you make this change you should see the input become re-focused after every time you click the "Add" or "Remove" buttons

**Review by dargaCode — CHANGES_REQUESTED** [review]
One tiny change to make the input refocus more often!

File: src/InputWithLabel.js (hunk 1 above)
Comment (Victoria240) [reply src/InputWithLabel.js]: Thanks a lot! I'll make the change and merge afterwards

**Pull request merged**